### PR TITLE
Intern Task Rita Bunets: Task homework4

### PR DESCRIPTION
**Pull request by ritabunets**

### CODE REVIEW

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: Homework4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+        //Task 5 - Display "Labas!" in case "Hello" word exist in the array.
+        private static void Task5()
+        {
+            string[] words = {"Privet", "Hey", "Hello", "Hola", "Coca-cola"};
+            string condWord = "Hello";
+            foreach (string word in words)
+            {
+                if (word == condWord)
+                Console.WriteLine("Labas!");
```
[the remark this answers is not in the document]
Comment (ritabunets) [reply Homework4/Program.cs:97]: fixed

File: Homework4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        private static void Task6()
+        {
+            int[] numbers = new int[5];
+            int sum;
+            Console.WriteLine("Enter 5 int numbers to create an array");
+            for(int i = 0; i < numbers.Length; i++)
+            {
+                numbers[i] = int.Parse(Console.ReadLine());
+            }
+            sum = numbers[0] + numbers[4];
```
[the remark this answers is not in the document]
Comment (ritabunets) [reply Homework4/Program.cs]: fixed

File: Homework4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        //Task 7 - Calculate the sum of the indexes of max and min elements
+        private static void Task7()
+        {
+            int[] numbers = new int[5];
+            Console.WriteLine("Enter 5 int numbers to create an array");
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                numbers[i] = int.Parse(Console.ReadLine());
+            }
+            int maxNumber = numbers.ToList().IndexOf(numbers.Max());
```
[the remark this answers is not in the document]
Comment (ritabunets) [reply Homework4/Program.cs]: fixed

File: Homework4/Program.cs [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                i++; 
+            }             
+            Console.WriteLine($"2048 should be divided {i} times by 2 to be less than 10.");
+        }
+
+        //Task 5 - Display "Labas!" in case "Hello" word exist in the array.
+        private static void Task5()
+        {
+            string[] words = {"Privet", "Hey", "Hello", "Hola", "Coca-cola"};
+            string condWord = "Hello";
```
Comment (yauhenists) [inline Homework4/Program.cs:93]: Not a mistake - usually such variables can be declared as constants

File: Homework4/Program.cs [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+                numbers[i] = int.Parse(Console.ReadLine());
+            }
+            int max = numbers[0];
+            int indexMax = 0;
+            int min = numbers[0];
+            int indexMin = 0;
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                if (max <= numbers[i])
+                {
```
Comment (yauhenists) [inline Homework4/Program.cs:132]: Here is the problem - you don't remember new max and min values when condition is met:
e.g. you have {3, 5, 4, 1, 2} - you will get indexMax = 2, indexMin = 4

**Review by yauhenists — CHANGES_REQUESTED** [review]

File: Homework4/Program.cs (hunk 5 above)
Comment (ritabunets) [reply Homework4/Program.cs:132]: Not sure I understand you correctly. 
1) Based on you example {3, 5, 4, 1, 2} - I will get indexMax =1, indexMin = 3. Am I right?
2) Why do I need to save max or min values? The task is related to their indexes. I do not need to save the value, I need to know its index. Is it correct?

File: Homework4/Program.cs (hunk 4 above)
Comment (ritabunets) [reply Homework4/Program.cs:93]: fixed

**Review by yauhenists — APPROVED** [review]

**Pull request merged**